### PR TITLE
Issue#8:Personal page added

### PR DESCRIPTION
**Pull request by bhavyagor12**
## Description

Added new folder for builders
Added personal address folder to have personal page
Made UI changes (avatar,bio etc)

UI showcase:
image

## Additional Information

- [x] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [x] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

## Related Issues
#8 

Your ENS/address:
0x95E08FA8ac4301acC5b943f860Cd8AC84433e3CF

### CODE REVIEW

**Comment by technophile-04** [comment]
Thanks @bhavyagor12 !! Looking great !! Just added a couple of small comments 🙌

File: packages/nextjs/app/builders/[0x95E08FA8ac4301acC5b943f860Cd8AC84433e3CF]/page.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+"use client";
+
+import type { NextPage } from "next";
+
+const PersonalPage: NextPage = () => {
+  return (
+    <div className="flex flex-col items-center md:flex-row justify-center md:h-[80vh] md:gap-4 ">
+      <div className="flex flex-col items-center mb-4 md:mb-0">
+        <img
```
Comment (technophile-04) [inline packages/nextjs/app/builders/[0x95E08FA8ac4301acC5b943f860Cd8AC84433e3CF]/page.tsx:9]: You could add : 

`{/* eslint-disable-next-line @next/next/no-img-element */}`

To fix gh-actions failing, we could have use Next Image component but then we need to add remote pattern to `next.config` maybe lets ignore it for now 🙌

**Comment by bhavyagor12** [comment]
> Thanks @bhavyagor12 !! Looking great !! Just added a couple of small comments 🙌

**Comment by technophile-04** [comment]
Lol my bad, They should be visible now 🙌

File: packages/nextjs/app/builders/[0x95E08FA8ac4301acC5b943f860Cd8AC84433e3CF]/page.tsx (hunk 1 above)
Comment (bhavyagor12) [reply packages/nextjs/app/builders/[0x95E08FA8ac4301acC5b943f860Cd8AC84433e3CF]/page.tsx]: ya i added it initially, to the next.config. but reverted it. If a lot of people in the batch are using it from the same source, I can add it back.

**Review by technophile-04 — APPROVED** [review]
Thanks !! Looks great !!

**Pull request merged**